package/yast2-iscsi-client.changes

```diff
@@ -12,6 +18,12 @@ Wed Jul  2 15:24:46 UTC 2025 - José Iván López González <jlopez@suse.com>
   bsc#1245171).
 - 5.0.7
 
+-------------------------------------------------------------------
+Fri Jan 31 15:53:50 UTC 2025 - Knut Alejandro Anderssen González <kanderssen@suse.com>
```

Comment:
Please move this entry just below yours, with an appropriate timestamp, so that we grow the changelog just at the top